packages/core/src/amazonq/webview/ui/main.ts

```diff
@@ -299,13 +341,19 @@ export const createMynahUI = (ideApi: any, featureDevInitEnabled: boolean, gumby
             }
 
             if (prompt.command !== undefined && prompt.command.trim() !== '') {
-                quickActionHandler.handle(prompt, tabID)
+                quickActionHandler.handle(prompt, tabID, eventId)
                 return
             }
 
             textMessageHandler.handle(prompt, tabID)
         },
         onVote: connector.onChatItemVoted,
+        onInBodyButtonClicked: (tabId, messageId, action, eventId) => {
+            connector.onCustomFormAction(tabId, messageId, action, undefined)
```

Comment:
NIT: Probably don't have to pass in undefined to the function if `undefined` is a type for that argument. Would only have to pass in `undefined` if there were more params after. Eg) `            connector.onCustomFormAction(tabId, messageId, action, undefined, anotherArgument)`

Reply:
Thanks for catching this - there's actually a different bug here where `eventId` was meant to be passed here but I failed to do so